digitalfabrik/entitlementcard
# 1211: Send DeepLinks via mail

### Short description

As a user i would like to receive a confirmation mail if my card was created by a service user to be informed about the current status. 

### Proposed changes



- add confirmation mail to user if card was created via application and a mail address was provided
- confirmation mail includes deep link
- pass mail address to `MailNotSendError`
- send mail as multipart to ensure better mobile mail client compatibility (created separate issue for adding html mail templates to other messages https://github.com/digitalfabrik/entitlementcard/issues/1374)

### Side effects



none

### Resolved issues



Fixes: #1211

### Testing
- go to applications and create a card (ensure the applicant has a mail address you can access) 
- check your mails
- open the mail on your smartphone and click activation link (works only on mail clients and browsers that support external app links)
- ensure config-local.yml exists with local mail configuration (dev only)
